Sources/Extensions/UIKit/UIStackViewExtensions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIStackViewExtensions.swift
+//  SwifterSwift-iOS
+//
+//  Created by Benjamin Meyer on 2/18/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if os(iOS) || os(tvOS)
+import UIKit
+    
+// MARK: - Initializers
+@available(iOS 9.0, *)
+public extension UIStackView {
+    
+    /// SwifterSwift: Initialize an UIStackView with an array of UIView and common parameters.
+    ///
+    ///     let stackView = UIStackView(views: [UIView(), UIView()], axis: .vertical)
+    ///
+    /// - Parameters:
+    ///   - views: The UIViews to add to the stack.
```

Comment:
Looks like `arrangedSubviews` was not updated in docstring 🤔

Reply:
Fakenews 🤣